azps-10.0.0/Az.DynatraceObservability/New-AzDynatraceMonitorSSOConfig.md

```diff
@@ -42,7 +42,7 @@ This command create a dynatrace SingleSignOn resource.
 ## PARAMETERS
 
 ### -AadDomain
-array of Aad(azure active directory) domains
+array of Microsoft Entra ID(Microsoft Entra ID) domains
```

Comment:
Duplicate in parentheses.

Reply:
Spacing issue, won't fix for now. We can do a preliminary update to correct parentheticals not preceded by a space so that these will get picked up properly during the actual run.